src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -435,6 +435,7 @@ export function CollapsibleNavGroupEnabled({
         <div
           className={classNames({
             'bottom-container': true,
+            'eui-xScroll': true,
```

Comment:
How will this look on browser-OS combos that show a scrollbar?

Reply:
I get your point, I thought the class will help to hide the scrollbar, but if not, let me find if there is any pattern code I can follow in OSD.

Reply:
Seem behavior of adding the utility css class follows the OS settings. Was the intention to hide the scrollbar but allow user to scroll?

Reply:
I added -ms-overflow-style: -ms-autohiding-scrollbar; to the bottom container to make the behavior of scrollbar consistent. Do you think that's good enough?

Reply:
I'm fine with it, UX need to figure out how to fit those overflow icons, I think we can optimize this afterwards.